config/config_test.go

```diff
@@ -3266,77 +3268,3 @@ func TestIsAccountEventEnabled(t *testing.T) {
 func boolPointer(b bool) *bool {
 	return &b
```

Comment:
Nitpick: To avoid repeated code in the codebase, please use the `ToPtr()` function found in `util/ptrutil/ptrutil.go`